lametro/templatetags/lametro_extras.py

```diff
@@ -176,12 +176,28 @@ def get_highlighted_attachment_text(context, id):
 @register.filter
 def matches_query(tag, request):
     # request.GET['q'] looks like "token AND token AND token"
-    return tag.lower() in [token.lower() for token in request.GET.get('q', '').split(' AND ')]
+    terms = [
```

Comment:
The changes to the template tags file highlight tags that both exactly and partially match search terms, e.g., searching "red line" will highlight "metro red line".